modules/upcloud.py

```diff
@@ -315,16 +315,17 @@ def main():
     api_user = module.params.get('api_user') or os.getenv('UPCLOUD_API_USER')
     api_passwd = module.params.get('api_passwd') or os.getenv('UPCLOUD_API_PASSWD')
     default_timeout = os.getenv('UPCLOUD_API_TIMEOUT')
+
     if not default_timeout:
-        default_timeout = module.params.get('upcloud', 'default_timeout') or None
```

Comment:
Why was this removed? Should the same default_timeout logic be here too (from above)? I'm not very familiar with the call structure here 😅

Reply:
These are ansible module params, since the user might use their upcloud account as server user, or define a new one. Timeout doesn't really play a part in the created server, but does in the call to API.